app/uk/gov/hmrc/nisp/controllers/AccountController.scala

```diff
@@ -58,6 +60,18 @@ trait AccountController extends NispFrontendController with AuthorisedForNisp wi
   val customAuditConnector: CustomAuditConnector
   val applicationConfig: ApplicationConfig
 
+  def showCope: Action[AnyContent] = AuthorisedByAny.async { implicit user => implicit request =>
+    isFromPertax.flatMap { isPertax =>
+      val nino = user.nino.getOrElse("")
+      val authenticationProvider = getAuthenticationProvider(user.authContext.user.confidenceLevel)
+      nispConnector.connectToGetSPResponse(nino).map {
+        case SPResponseModel(Some(spSummary: SPSummaryModel), None) =>
```

Comment:
This needs to check if they can see this with `getABTest` and `Redirect` back to the account page if they can't.